src/AbstractHyperrectangle.jl

```diff
@@ -92,13 +95,37 @@ Return the norm of a hyperrectangular set.
 
 A real number representing the norm.
 
-### Notes
+### Algorithm
 
-The norm of a hyperrectangular set is defined as the norm of the enclosing ball,
-of the given ``p``-norm, of minimal volume that is centered in the origin.
+Recall that the norm is defined as
+
+```math
+‖ X ‖ = \\max_{x ∈ X} ‖ x ‖_p = max_{x ∈ \\text{vertices}(X)} ‖ x ‖_p.
+```
+The last equality holds because the optimum of a convex function over a polytope
+is attained at one of its vertices.
+
+This implementation uses the fact that the maximum is achieved in the vertex
+``c + \\text{diag}(\\text{sign}(c)) r``, for any ``p``-norm, hence it suffices to
+take the ``p``-norm of this particular vertex. This statement is proved below.
+Note that, in particular, there is no need to compute the ``p``-norm for *each*
+vertex, which can be very expensive. 
+
+If ``X`` is an axis-aligned hyperrectangle. If the ``n``-vectors center and radius
```

Comment:
Remove the period and merge the two sentences: `... hyperrectangle and (if) ...`